### DISCUSSION TITLE
Building a pc

### DISCUSSION THREAD

**Discussion by akariwrynn** [pybamm-team/PyBaMM#3314] [General]
Hello,
My advisor wants me to build a pc that can run python (especially pybamm) in a fast manner. For this purpose, I thought I should choose a motherboard with high clock speed (better cpu speed as I thought pybamm is running on cpu rather than gpu). Besides that, what can you suggest for a pc that will run highly cpu demanding codes?

P.S: I am a chemistry student so I dont have a strong background about building a pc.

Thanks a lot

**Comment by brosaplanella** [comment]
Hi! PyBaMM focuses on the software, so unfortunately we can't provide input on hardware as this is out of our area of expertise.
